fuse/readonly/readonly_unix.go

```diff
@@ -62,13 +62,19 @@ func (s *Root) Lookup(ctx context.Context, name string) (fs.Node, error) {
 		return nil, syscall.Errno(syscall.ENOENT)
 	}
 
-	p, err := path.ParsePath(name)
+	p, err := path.NewPath(name)
```

Comment:
@hacdias I think this change might've broken fuse ipfs mount (ipns still works, I think):

```
2023-11-28T14:01:52.654+0100    DEBUG   fuse/ipfs       readonly/readonly_unix.go:67    fuse failed
 to parse path: "QmREP1hYvkmdfHN3ybouWFaB7BenoinYRhWgcMwVWdutw4": invalid path "QmREP1hYvkmdfHN3ybo
uWFaB7BenoinYRhWgcMwVWdutw4": path does not have enough components
```

Reply:
It is broken beyond that though, adding `/ipfs/` prefix results in:

```
2023-11-28T14:14:19.976+0100    DEBUG   fuse/ipfs       readonly/readonly_unix.go:58    Root Lookup: 'Qmde3RkqLihpS2ehi8VqUm6gZ9dCVGg4AvqGRcK5uvt78z'
2023-11-28T14:14:19.977+0100    ERROR   fuse/ipfs       readonly/readonly_unix.go:113   could not convert protobuf or raw node: expected protobuf dag node
```

Reply:
@crackcomm thanks for pinging. We are trying to take a look at it in https://github.com/ipfs/kubo/pull/10243.